src/main/java/view/InputViewImpl.java

```diff
@@ -0,0 +1,39 @@
+package view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+import message.RequestMessage;
+import validation.Validation;
+
+public class InputViewImpl implements InputView {
+
+  private final Validation validation = new Validation();
+
+  @Override
+  public List<String> inputCar() throws IllegalArgumentException {
+    System.out.println(RequestMessage.REQUEST_CARS);
+    String inputCars = Console.readLine();
+    List<String> cars = carNameSeparator(inputCars);
+    validation.validateCharacterCount(cars);
```

Comment:
이건 약간 취향인듯? 입력의 책임을 벗어난거 같음
```suggestion
    validation.validateCharacterCount(cars);
```